src/main/java/ch/njol/skript/lang/SkriptParser.java

```diff
@@ -263,7 +261,14 @@ private <T extends SyntaxElement> T parse(Iterator<? extends SyntaxElementInfo<?
 								x = x2;
 							}
 							T t = info.c.newInstance();
-							if (t.init(res.exprs, i, getParser().getHasDelayBefore(), res)) {
+							List<TriggerSection> sections = getParser().getCurrentSections();
+							TriggerSection current = sections.isEmpty() ? null : sections.get(sections.size() - 1);
+							if (t.getUsableEvents().length > 0 && !CollectionUtils.containsAny(t.getUsableEvents(), getParser().getCurrentEvents()))
```

Comment:
In an else if statement, we still include braces around the statements as it doesn't increase line size and is still readable.

Reply:
Makes sense, added !